server/datastore/mysql/migrations/tables/20210601000008_TeamsEnrollSecrets.go

```diff
@@ -71,6 +71,7 @@ func Up_20210601000008(tx *sql.Tx) error {
 		ALTER TABLE enroll_secrets
 		DROP COLUMN active,
 		DROP COLUMN name,
+		ADD PRIMARY KEY (secret),
 		ADD UNIQUE INDEX (secret)
```

Comment:
This becomes unnecessary once it's a primary key.

Reply:
> This becomes unnecessary once it's a primary key.

Agreed, but I just wanted to be sure and have you guys take a look anyways. 🦖